_layouts/default.html

```diff
@@ -36,6 +36,11 @@
             <li class="nav__item nav__item--header">
               <a class="nav__link nav__link--header" href="http://status.siteleaf.com/">Status</a>
             </li>
+            {% if page.nav_toggle != false %}
+              <li class="nav__item nav__item--header nav__toggle">
+                <span class="nav__link nav__link--header">&#9776;</span>
```

Comment:
This button should probably be a `<button>` element. Besides being more semantically correct, it's more accessible and I _think_ gives you friendlier button interactions on phones, for free.